testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDropTableCompatibility.java

```diff
@@ -61,7 +61,8 @@ public static Object[][] engineConfigurations()
                 {TRINO, DELTA, false},
                 {DELTA, TRINO, true},
                 {DELTA, DELTA, true},
-                {DELTA, DELTA, false},
+                {DELTA, TRINO, false},
+                {DELTA, DELTA, false}
```

Comment:
it's resolved, but the comma is missing :)

Reply:
I felt there is no more need to add `,` to support adding eventually in the future another combination, because all the engine configurations are already listed. Nevertheless, I'm adding the extra `,`  to be in sync with the existing code in Trino.